esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -146,17 +158,15 @@ IPropertyTree * fetchConfigInfo(const char * config,
             deftree = configTree->queryBranch("Definition[1]");
 
         if (deftree)
-        {
-            methodstree = deftree->getBranch("Methods");
-        }
+            retTree = deftree->getBranch(treeName.get());
 
-        if (!methodstree) //if we didn't already find the methods section of the config, let's look at the root
+        if (!retTree) //if we didn't already find the methods section of the config, let's look at the root
```

Comment:
Update comment to not be methods-centric.

Reply:
will fix